clients/fides-js/src/fides-tcf.ts

```diff
@@ -136,7 +141,8 @@ const updateCookie = async (
   };
 
   const tcString = await generateTcString({ tcStringPreferences, experience });
-  return { ...oldCookie, tc_string: tcString };
+  const tcfConsent = transformTcfPreferencesToCookieKeys(tcStringPreferences);
+  return { ...oldCookie, tc_string: tcString, tcf_consent: tcfConsent };
```

Comment:
this probably should have been in https://github.com/ethyca/fides/pull/4124 but I didn't realize it was missing until I started switching between TCF and non-TCF